.cursor/commands/pipeline.md

```
@@ -380,44 +152,44 @@ For **multi-phase modes** (full, design-and-review, implement):
 <important architectural or implementation decisions made during the pipeline>
 ```
 
-For **single-phase modes** (design-only, review-only, arch-review, code-review-only, qa-only):
+**For single-phase modes** (`design-only`, `review-only`, `arch-review`, `code-review-only`, `qa-only`):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)